File: r.py
import math

def seeded_random(seed, min, max):
    l_seed = seed
    def rd():
        nonlocal l_seed
        random = math.sin(l_seed) * 10000
        seed_decimal = random - math.floor(random)
        l_seed = math.floor(seed_decimal * 2147483647)
        return math.floor(seed_decimal * (max - min + 1) + min)
    return rd

seed = 12345
min = 1
max = 100
r = seeded_random(seed, min, max)
for i in range(100):
    print(r())

File: r.js
function seededRandom(seed, min, max) {
    var l_seed = seed;
    function rd() {
        const random = Math.sin(l_seed) * 10000;
        const seedDecimal = random - Math.floor(random);
        l_seed = Math.floor(seedDecimal * 2147483647);
        return Math.floor(seedDecimal * (max - min + 1) + min);
    }
    return rd;
}

const seed = 12345;
const min = 1;
const max = 100;
const r = seededRandom(seed, min, max);
for (let i = 0; i < 100; i++) {
    console.log(r());
}